docs/src/pages/components/tabs/tabs.md

```diff
@@ -171,3 +171,29 @@ Focus a tab and navigate with arrow keys to notice the difference, e.g. <kbd cla
 ```
 
 {{"demo": "pages/components/tabs/AccessibleTabs2.js", "defaultCodeOpen": false}}
+
+## Unstyled
+
+The Tabs also come with an unstyled version. It's ideal for doing heavy customizations and minimizing bundle size.
+
+### Unstyled component
+
+```js
+import TabsUnstyled from '@mui/base/TabsUnstyled';
+import TabsListUnstyled from '@mui/base/TabUnstyled';
+import TabUnstyled from '@mui/base/TabUnstyled';
+import TabPanelUnstyled from '@mui/base/TabPanelUnstyled';
+```
+
+{{"demo": "pages/components/tabs/UnstyledTabsBasic.js"}}
```

Comment:
Seeing this demo makes me wonder about how we want to demo unstyled. Did we settle on something in the past?  I can't remember.

Reply:
I think that what Marija did was great: showcasing how the component looks if you just import it in your code and add no styling at all - native HTML elements. And then, a customization example. I think this can be a rule of thumb for the unstyled. Do you envision something else?

Reply:
I'm asking because we use different approaches in the docs. For instance, Tabs + Autocomplete + Modal are very much into the first demo is as bare as possible, the second demo looks pretty.
The other unstyled's components demo doesn't try to look bare.

Reply:
Got it. I think it's helpful to have it as we have here in the Tabs, a bare feet example and then customized.